joint_trajectory_controller/src/joint_trajectory_controller_parameters.yaml

```
@@ -44,15 +44,15 @@ joint_trajectory_controller:
     default_value: false,
     description: "Allow joint goals defining trajectory for only some joints.",
   }
-  open_loop_control: {
+  interpolate_from_desired_state: {
     type: bool,
-    default_value: false,
-    description: "Use controller in open-loop control mode
+    default_value: true,
```

we should leave this default to false, otherwise we change the behavior of existing setups
```suggestion
    default_value: false,
```